ui2.0/app/actions/password.js

```diff
@@ -0,0 +1,11 @@
+import { SET_REENTER } from 'constants/password';
+
+export const setReenter = () => ({
```

Comment:
if you use a fixed payload value, I suggest to call it `showReenter`(in the opposite of `hideReenter`)